### PR TITLE
WASM: Handle global simd correctly.

### PR DESCRIPTION
**Pull request by lizhengxing**
Add explicit checks for handing global simd correctly.

### CODE REVIEW

**Comment by lizhengxing** [comment]
@Cellule 
Hi, Michael:
This is the initial PR for WASM: Global SIMD incorrectly handled #5078 , As Arun is on vacation, I sent it out to you guys directly.

Please help to review it if you are available, thanks!

--zhengxing

File: lib/Runtime/Library/WebAssemblyInstance.cpp [hunk 1]
```diff
@@ -257,9 +257,21 @@ Var WebAssemblyInstance::CreateExportObject(WebAssemblyModule * wasmModule, Scri
             case Wasm::ExternalKinds::Memory:
                 obj = env->GetMemory(wasmExport->index);
                 break;
-            case Wasm::ExternalKinds::Function:
+            case Wasm::ExternalKinds::Function: {
                 obj = env->GetWasmFunction(wasmExport->index);
+                // Check whether there's any SIMD value in the function's signature.
+                Wasm::WasmSignature* sig = ((WasmScriptFunction*)(obj))->GetSignature();
```
Comment (Cellule) [inline lib/Runtime/Library/WebAssemblyInstance.cpp]: It is valid to export a function with simd arguments/return.
If that function is called from javascript then we should throw.
However, if that function is then imported in another wasm module it is fine to use.
In general, the handling of simd types in functions is fine as it is today. (unless it was spec'ed that way, but I don't see why)

**Comment by Cellule** [comment]
By looking at this PR I realized that I had a few concerns with the proposal of how to handle simd in javascript in the proposal.
We should wait until we resolve that in the proposal before moving on with this PR I think

File: lib/Runtime/Library/WebAssemblyInstance.cpp (hunk 1 above)
Comment (lizhengxing) [reply lib/Runtime/Library/WebAssemblyInstance.cpp]: @Cellule 
Thanks for your comment. As arun is still on vacation, Would you please pend this issue and discuss with him when he's back?

Thanks!

**Comment by aretmr** [comment]
@Cellule Has this been addressed through your commit https://github.com/Microsoft/ChakraCore/commit/ad1f74e17a1f5f5735f53f10add309e745729bae ? If so shall we close this PR?

**Comment by Cellule** [comment]
The handling I have done is not complete however. I made sure that we don't crash by refusing simd globals.
There should be more work done to make simd globals available and behave like int64.
I might look into it when I'll work on WebAssembly.Global (aka mutable global) feature.
As for this particular PR, it probably can be closed as its content doesn't reflect what we want for simd globals

**Comment by Cellule** [comment]
Closing this for now as it will need some rework to correctly support wasm.simd globals

**Pull request closed without merging**